src/util/wren2c.c

```diff
@@ -0,0 +1,81 @@
+
+internal 
+void WREN2C_randomString(char *dest) {
+    // https://stackoverflow.com/a/15768317
+    char charset[] = "0123456789"
+                     "abcdefghijklmnopqrstuvwxyz"
+                     "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+    srand(time(0));
+
+    size_t length = 10;
+    while (length-- > 0) {
+        size_t index = (double) rand() / RAND_MAX * (sizeof charset - 1);
+        *dest++ = charset[index];
+    }
+    *dest = '\0';
+}
+
+
+internal 
+void WREN2C_encodeAndDump(ENGINE* engine, int argc, char* args[]) {
+  
+  char* fileName = args[2];
+  char* fileOut = strcat(strdup(fileName), ".inc");
+
+  char randName[] = "";
+  WREN2C_randomString(randName);
+
+  char* moduleName = randName;
```

Comment:
It would be better to only compute a alternate/default name iff the moduleName wasn't specified. I'm not certain it should be randomised, but a default is good for testing.

Reply:
Having the example https://github.com/wren-lang/wren/blob/main/util/wren_to_c_string.py it uses the filename as the default module name. Maybe something similar can be used for a default name. But i dont know how complex can be done that in C.